.storybook/preview.ts

```diff
@@ -2,12 +2,15 @@ import type { Preview } from "@storybook/react";
 import * as VibeComponents from "monday-ui-react-core";
 import * as VibeNext from "monday-ui-react-core/next";
 import * as VibeIcons from "monday-ui-react-core/icons";
+import reactDocgenOutput from "./react-docgen-output-example.json";
```

Comment:
Why is it an example? is it auto-generated?

Reply:
it is part of the `react-docgen` json output run on `monday-ui-react-core`, for example purposes